lib/xlat_tables_v2/xlat_tables_internal.c

```diff
@@ -1060,7 +1060,7 @@ void init_xlation_table(xlat_ctx_t *ctx)
 #if PLAT_XLAT_TABLES_DYNAMIC
 		ctx->tables_mapped_regions[j] = 0;
 #endif
-		for (int i = 0; i < XLAT_TABLE_ENTRIES; i++)
+		for (uint32_t i = 0; i < XLAT_TABLE_ENTRIES; i++)
```

Comment:
It looks odd that this becomes unsigned but the other loop counters in this function remain signed. It looks like they should also become unsigned.